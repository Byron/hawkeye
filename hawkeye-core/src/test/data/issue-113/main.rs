use macros::define_result;
