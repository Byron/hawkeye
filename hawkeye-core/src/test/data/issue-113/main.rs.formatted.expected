// main.rs
// ------------------------------------------
// Copyright (c) 2024
// Some_Name
// ------------------------------------------

use macros::define_result;
